assets/package.json

```diff
@@ -14,16 +16,46 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "@vue/test-utils": "^1.0.0-beta.30",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.0",
     "babel-preset-vue": "^2.0.2",
     "copy-webpack-plugin": "^4.5.0",
     "css-loader": "^2.1.1",
+    "jest": "^24.9.0",
     "mini-css-extract-plugin": "^0.4.0",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "prettier": "^1.19.1",
     "terser-webpack-plugin": "^1.1.0",
+    "vue-jest": "^3.0.5",
     "vue-loader": "^15.7.2",
     "vue-template-compiler": "^2.6.10",
     "webpack": "4.4.0",
     "webpack-cli": "^3.3.2"
+  },
+  "jest": {
+    "modulePaths": [
+      "<rootDir>"
+    ],
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/js/$1"
+    },
+    "transform": {
+      "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
+      ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest"
+    },
+    "coverageDirectory": "<rootDir>/js/test/unit/coverage",
+    "collectCoverageFrom": [
+      "<rootDir>/js/components/**/*.{js,vue}",
+      "!<rootDir>/js/components/_components.js",
+      "!<rootDir>/js/components/_lux-elements-used.js",
+      "!<rootDir>/node_modules/**"
+    ]
   }
 }
```

Comment:
@jrgriffiniii I was probably unclear in a previous PR comment, but I liked how you had these `jest` configs in the previous jest config file.  What I meant to say was that we might not (not positive) need to declare the path to `jest.config.js` or whatever the file is called, in the `test` npm command.   

Either way, no big deal.  Tested, and locally runs as expected.  Again, nice work.  I'll save my work on top of this and do a PR in the near future to expand some testing scenarios with embedded components and other more advanced scenarios which will pop up in a future app.

Reply:
Apologies for that, I have restored this with https://github.com/samvera-labs/digital_collections_elixir_example/pull/33/commits/92ca869b2d4312b90ee79443d10b95c07e421f18